__tests__/components/sticky_header_test.js

```diff
@@ -37,20 +37,20 @@ describe("StickyHeader", () => {
     ).toContain("0");
   });
 
-  it("contains edit selections link", () => {
-    expect(
-      mount(<StickyHeader {...props} />)
-        .find("#editSelections")
-        .first().length
-    ).toEqual(1);
-  });
+  // it("contains edit selections link", () => {
```

Comment:
The changes in this test are only temporary until we remove this sidebar post-beta.